### ISSUE TITLE
button clicked but nothing happen after switching to http

### ISSUE THREAD

**Issue by ijash** [SchizoDuckie/PimpMyuTorrent#14]
as the title says while trying to pimp the utorrent

**Comment by garfield69** [comment]
broken with uTorrent 3.5.3
https://engineering.bittorrent.com/2018/02/22/httprpc-security-vulnerabilities-resolved-in-utorrent-bittorrent-and-utorrent-web/

**Comment by garfield69** [comment]
@ijash  as a possible circumvention, you might like to give this a go:
1. install a version of uTorrent prior to 3.5.3
2. use the pimp to set the parms
3. install uTorrent 3.5.3 (without un-installing the prior version)
4. this should just upgrade uTorrent to 3.5.3 while keeping your pimped settings.

**Comment by Wolivere** [comment]
Dont need to install anything. Go to webui and set your listening port to 10000 from 8080.

Then try the button you should get the pop up. After your pimped, then set your listening port to what ever you want.

**Comment by garfield69** [comment]
@Wolivere  fantastic! thanks for the info.

**Comment by n6432674** [comment]
changing listening port to 10000 made utorrent show the pop up but still wont remvoe ads. tried installing an older version too but didnt work as well

**Comment by garfield69** [comment]
did you restart utorrent after pimping, to pick the new settings ?

**Comment by n6432674** [comment]
i did restart but the ads are still there

**Comment by garfield69** [comment]
strange. I have tested both methods described in this ticket, and had no problems pimping utorrent.
tested with 3.5.1 and 3.5.4 on my windows platform.

**Comment by n6432674** [comment]
i tried with older versions as well with no luck....

**Comment by garfield69** [comment]
ok, take a look at the console logs to see the activity behind the scenes.
1. install the latest utorrent for windows https://www.utorrent.com/downloads/complete/track/stable/os/win
2. start utorrent and set the webui port to 10000
![image](https://user-images.githubusercontent.com/6933240/45942105-3b91a300-c035-11e8-845a-227058493f66.png)
3. reboot your system (might as well go hardcore and ensure everything is refreshed ;-)
4. navigate your chrome browser to the pimpmytorrent http://schizoduckie.github.io/PimpMyuTorrent/#/
5. press the **F12** key to open the *development tools* page, and click on the **console** tab to view the log
6. click on the **pimp my utorrent** button on the pimpmytorrent page
7. click **OK** to the *allow access to utorrent* popup that should be presented by utorrent
8. view the log and look for activity.

here is an example of a successful pimp log
![image](https://user-images.githubusercontent.com/6933240/45941907-62030e80-c034-11e8-999c-2f06cc8b5106.png)

what do you get?

**Comment by n6432674** [comment]
thx!! will tell you

**Comment by n6432674** [comment]
i have to look at the chrome log and not utorrent log?

the f12 chrome log only says:

?from=https:92 GET https://schizoduckie.github.io/PimpMyuTorrent/img/bgnoise.png 404

when clicking the button

**Comment by garfield69** [comment]
yes, once you are on the pipmyutorrent page, use the **F12** key.
the button you press is this one
![image](https://user-images.githubusercontent.com/6933240/45942726-ae9c1900-c037-11e8-8603-5a379a93f8f7.png)
and you will only get a couple of lines of activity on the log until you answer the utorrent popup to allow access
![image](https://user-images.githubusercontent.com/6933240/45942835-2bc78e00-c038-11e8-86d0-9ce57a352799.png)
if you accidentally replied no, then the log should show **pair denied**
![image](https://user-images.githubusercontent.com/6933240/45942870-51549780-c038-11e8-80ab-d338472d7f29.png)

**Comment by garfield69** [comment]
Oh, and if you have already used pipmytorrent once, then use the refresh key on your browser to start it up again.
I just noticed that clicking the button twice does not start the connection!! :-)

**Comment by n6432674** [comment]
i did press yes and tried refreshing too but all i get is this:

http://prntscr.com/kxz9em

**Comment by n6432674** [comment]
thx for helping out so far

**Comment by garfield69** [comment]
well i don't know what is going on with your browser and pimpmytorrent.

BUT, you can still do the pimping, manually, if you want. its not difficult.
access the advanced preferences for utorrent and set the following:

filter by *offers* and set them all *false*
![image](https://user-images.githubusercontent.com/6933240/45943450-721dec80-c03a-11e8-8581-24cefca5bbe4.png)

just select each entry and click on the false radio button
![image](https://user-images.githubusercontent.com/6933240/45943474-8a8e0700-c03a-11e8-96fd-75c1bf4ed67d.png)

filter by *show* and set them all false
![image](https://user-images.githubusercontent.com/6933240/45943520-b7421e80-c03a-11e8-82af-843d0d2607e6.png)

there is a couple more, the full list to set is in the first log i showed you earlier :-)

**Comment by n6432674** [comment]
oh wait it worked once i used a different browser sweet, i have alot of extensions so that might cause it to not work

**Comment by garfield69** [comment]
congrats.

**Comment by spawn-guy** [comment]
@garfield69 a very nice condensed tutorial! I've followed a different one, but it did't "pimp".
after "offers" and "show" searches `uT` started to look as I want it. thanks.

would you, maybe, like to add this to the "README.md" as a "manual" alternative way of pimping?

**Comment by garfield69** [comment]
Sorry, I don't have admin rights to this repo, so I can not close tickets or merge PRs.
you may have noticed that there are issues and PRs that have not progressed for over a year now.
I can submit a PR with the updated readme, but since schizoduckie isn't monitoring this site, the PR will linger un-merged.
So its not worth my time.

**Comment by ijash** [comment]
> ok, take a look at the console logs to see the activity behind the scenes.
> 
> 1. install the latest utorrent for windows https://www.utorrent.com/downloads/complete/track/stable/os/win
> 2. start utorrent and set the webui port to 10000
>    ![image](https://user-images.githubusercontent.com/6933240/45942105-3b91a300-c035-11e8-845a-227058493f66.png)
> 3. reboot your system (might as well go hardcore and ensure everything is refreshed ;-)
> 4. navigate your chrome browser to the pimpmytorrent http://schizoduckie.github.io/PimpMyuTorrent/#/
> 5. press the **F12** key to open the _development tools_ page, and click on the **console** tab to view the log
> 6. click on the **pimp my utorrent** button on the pimpmytorrent page
> 7. click **OK** to the _allow access to utorrent_ popup that should be presented by utorrent
> 8. view the log and look for activity.
> 
> here is an example of a successful pimp log
> ![image](https://user-images.githubusercontent.com/6933240/45941907-62030e80-c034-11e8-999c-2f06cc8b5106.png)
> 
> what do you get?

Works! popup appears. This one works for me. thank you

**Comment by SeaHOH** [comment]
Go #19 to have a try.
